packages/xo-web/src/common/intl/messages.js

```diff
@@ -886,7 +886,7 @@ const messages = {
   vdiDisconnect: 'Disconnect',
   vdiTags: 'Tags',
   vdiSize: 'Size',
-  vdiSr: 'SR',
+  vdiSr: 'SR container',
```

Comment:
Why?

Reply:
to change `SR` label to `SR container` in  `Health: Orphaned VDIs/ VDIs attached to Control Domain`  and `Vm/Disks`

Reply:
- But those views actually show the SR's name, not its container's, right?
- When we change the meaning of a message would should either make sure that all the locales are correctly updated to the new meaning or create a new message (which is probably better)